apps/shadcn-example/.eslintrc.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
I think I had a comment on that before, but any reason we dont simply include it in the existing demo dapp instead of creating a new example?

Reply:
I figured that having a standalone shadcn example would be easier to understand from a user standpoint if they just want to add a wallet selector to their own shadcn app. Also, I would have to make a decent amount of changes to the existing demo dapp to add the light/dark mode switcher, and I wasn't sure if adding tailwind and all the shadcn components would make the dapp seem too busy.

Let me know what you think though. I probably could still merge them together.

Reply:
that makes sense. the idea for the demo dapp is 1) show users how they can use the adapter and 2) have a live site users can go to and see how everything works https://aptos-labs.github.io/aptos-wallet-adapter/

We can either have it all in one site or maybe have 2 different demo sites (which seems to much imo)

Reply:
That sounds fair. I'll re-style the deployed example dapp with shadcn and then delete the one I created. The MUI and AntD buttons will still stay obviously.

Reply:
I just pushed a commit to merge the two examples together.